ci/nightly/pipeline.yml

```diff
@@ -619,16 +619,17 @@ steps:
           composition: sqlancer
           args: [--runtime=3300, --oracle=QUERY_PARTITIONING]
 
-  - id: sqlancer-having
-    label: "SQLancer Having"
-    artifact_paths: junit_*.xml
-    timeout_in_minutes: 58
-    agents:
-      queue: linux-x86_64
-    plugins:
-      - ./ci/plugins/mzcompose:
-          composition: sqlancer
-          args: [--runtime=3300, --oracle=HAVING]
+  # TODO(def-) Reenable when #18346 is fixed
```

Comment:
I now see that buildkite has a `skip` annotation - https://buildkite.com/docs/pipelines/command-step that seems to have better ergonomics than commenting out.

skip | Whether to skip this step or not. Passing a string provides a reason for skipping this command. Passing an empty string is equivalent to false. Note: Skipped steps will be hidden in the pipeline view by default, but can be made visible by toggling the 'Skipped jobs' icon.Example: trueExample: falseExample: "My reason"
-- | --


Note however that #18346 being a philosophical issue of sorts, is unlikely to get fixed soon. It may be better to fix sqlancer instead to not output any of the questionable expressions.